packages/core/src/components/dialog/_dialog-body.scss

```diff
@@ -0,0 +1,13 @@
+.#{$ns}-dialog-body {
+  flex: 1 1 auto;
+  padding: $dialog-padding;
```

Comment:
Changed margin to padding so the scroll works better. Should not cause regressions.
Removed line-height, did not seem appropriate default.